views/membership.py

```diff
@@ -1,3 +1,5 @@
+from rest_framework import status
+from rest_framework.response import Response
```

Comment:
Import `response` from DRF and use `response.Response` wherever. Makes imports much cleaner.